src/app/helpers/json/deepGet.test.js

```diff
@@ -0,0 +1,22 @@
+import deepGet from './deepGet';
+
+const fixtureData = {
+  foo: 123,
+  bar: {
+    baz: 456,
+  },
+};
+
+describe('deepGet', () => {
+  it('should retrieve items one level deep', () => {
+    expect(deepGet(['foo'], fixtureData)).toEqual(123);
+  });
+
+  it('should retrieve items two levels deep', () => {
+    expect(deepGet(['bar', 'baz'], fixtureData)).toEqual(456);
+  });
+
+  it('should return null for missing properties', () => {
```

Comment:
Reallly nit picky but can you add a test where the invalid key is deeper please?